src/utils/conversion.js

```diff
@@ -0,0 +1,99 @@
+export function convertAction(action, newProperties) {
+  const newAction = {
+    actionType: action.actionType,
+    bookmark: action.selectedBookmark,
+    // Will need to convert field selected by expression
```

Comment:
Are you planning to do this in this PR?

Reply:
Not really. This depends on how the new component will handle things as well I guess

Reply:
Oh ok, then it can wait yes